mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -20,19 +20,21 @@ pipeline {
         stage('Build MIOpenDriver') {
             steps {
                 dir('MIOpen') {
+                    sh 'mkdir install'
                     git branch: 'develop', poll: false,\
                                 url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
-                    cmake arguments: "-P install_deps.cmake --minimum",\
+                    cmake arguments: "-P install_deps.cmake --minimum --prefix ./MIOpen_deps",\
                                 installation: "InSearchPath"
                     cmakeBuild buildDir: 'build',\
                                 buildType: 'Release',\
                                 installation: 'InSearchPath',\
                                 cmakeArgs: '''
                                  -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
                                  -DMIOPEN_BACKEND=HIP
-                                 -DCMAKE_PREFIX_PATH="/opt/rocm/hip;/opt/rocm/;/usr/local" ..
+                                 -DCMAKE_PREFIX_PATH="${WORKSPACE}/MIOpen/MIOpen_deps"
+                                 -DCMAKE_INSTALL_PREFIX=../install/ ..
```

Comment:
What's the final `..` there for? I'll also note that it might make more sense to go with `${WORKSPACE}/MIOpen/install`.

Reply:
. . should not be needed as we have specified the buildDir